sdk/plugin/pb/backend.proto

```diff
@@ -403,6 +403,12 @@ message TypeReply {
 	uint32 type = 1;
 }
 
+// VersionReply is the reply for the Version method.
+message VersionReply {
+	string name = 1;
```

Comment:
`name` feels a bit too ambiguous to me, is this the plugin name? Maybe worth being a bit more verbose?

```suggestion
	string plugin_name = 1;
```